### PR TITLE
Change blurbs

### PR DESCRIPTION
**Pull request by sannecottaar**
Consistent message at the top of files. Also included Bob in the technical reference on the website.

### CODE REVIEW

**Comment by ian-r-rose** [comment]
Thanks, Sanne.  Can you squash?

**Comment by bobmyhill** [comment]
Thanks Sanne!
(Also, can I be Myhill, R., please?)

**Comment by sannecottaar** [comment]
Check and check.

File: burnman/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# BurnMan - a lower mantle toolkit
-# Copyright (C) 2012, 2013, Heister, T., Unterborn, C., Rose, I. and Cottaar, S.
-# Released under GPL v2 or later.
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
```
Comment (tjhei) [inline burnman/__init__.py]: fancy!

**Pull request merged**